client/my-sites/plans-features-main/index.jsx

```diff
@@ -294,6 +297,18 @@ export class PlansFeaturesMain extends Component {
 			</div>
 		);
 	}
+
+	filterDotBlogDomains() {
+		const domains = get( this.props, 'domains', [] );
+		return domains.filter( function( domainInfo ) {
+			if ( domainInfo.type === 'WPCOM' ) {
+				return false;
+			}
+
+			const domainName = get( domainInfo, [ 'domain' ], '' );
+			return ! 'blog'.startsWith( getTld( domainName ) );
+		} );
```

Comment:
I wonder if this would be a good opportunity to set `disableBloggerPlanWithNonBlogDomain` in `PlansStep ` with a proper selector, and include these changes there? That would let you avoid having to go the `nonDotBlogDomains` route.

Reply:
I did start there, but `PlansStep` is only used in signup so it seemed like this was the place for it to land.